src/Compadre_Evaluator.hpp

```diff
@@ -412,35 +413,37 @@ class Evaluator {
         }
 
 
-        // only written for up to rank 1 to rank 1 (in / out)
+        // only written for up to rank 1 to rank 2 (in / out)
         // loop over components of output of the target operation
-        for (int i=0; i<output_dimension_of_operator; ++i) {
+        for (int i=0; i<output_dimension1_of_operator; ++i) {
             const int output_component_axis_1 = i;
-            const int output_component_axis_2 = 0;
-            // loop over components of input of the target operation
-            for (int j=0; j<input_dimension_of_operator; ++j) {
-                const int input_component_axis_1 = j;
-                const int input_component_axis_2 = 0;
-
-                if (loop_global_dimensions) {
-                    for (int k=0; k<global_dimensions; ++k) { // loop for handling sampling functional
+            for (int axes1=0; axes1<output_dimension2_of_operator; ++axes1) {
```

Comment:
Should this be named axes2?

Reply:
yes, make i to axes1 and this axes1 to axes2 should be good.

Reply:
Thanks for the info!